argocd/overlays/moc-infra/alerts.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: argocd-alerts
+spec:
+  groups:
+    - name: argocd-alerts
+      rules:
+        - alert: ArgoAppSyncFailed
+          expr: argocd_app_sync_total{phase!="Succeeded"} == 1  # Triggered when argocd-application is not succeeded
```

Comment:
@mubariskhan96 can you add a description for this alert? I don't know how this metric works.

Reply:
added a descripton!